sdk/search/Azure.Search.Documents/src/Models/SearchResults{T}.cs

```diff
@@ -56,7 +56,7 @@ public class SearchResults<T>
         /// <summary>
         /// Gets the first (server side) page of search result values.
         /// </summary>
-        internal IList<SearchResult<T>> Values { get; } = new List<SearchResult<T>>();
+        internal IList<SearchResult<T>> Values { get; set; } = new List<SearchResult<T>>();
```

Comment:
It may not matter if this is only ever instantiated and assigned to once, but you might instead declare this property as a `List<SearchResult<T>>` and use it's `AddRange` method to copy the elements instead.